regen_plugins_cache.py

```diff
@@ -0,0 +1,2 @@
+from twisted.plugin import IPlugin, getPlugins
```

Comment:
I take it this is to fix the weird plugin loading thing. Please add comments here so nobody removes it in the future. Also, why not add this to the `__init__.py`?

Reply:
Included running this script in `run_all_tests_unix.py`